### PR TITLE
Add "Not found" page

### PR DESCRIPTION
**Pull request by rubo**
- Implemented "no match" route handler to display a dedicated page for not found URLs
- Revised layout to expand the content to screen size

### CODE REVIEW

File: src/shared/NotFound.jsx
Comment (vincenthongzy) [inline src/shared/NotFound.jsx:1]: image

on mobile view, does not happen with other pages

Comment (rubo) [reply src/shared/NotFound.jsx:1]: Fixed

Comment (vincenthongzy) [reply src/shared/NotFound.jsx:1]: the oversized twitter and discord logo will be fixed later yeah?

**Review by vincenthongzy — APPROVED** [review]

File: src/shared/NotFound.jsx
Comment (rubo) [reply src/shared/NotFound.jsx:1]: That's a good observation. Missed it somehow. Yes, next PR.

**Pull request merged**